resource/traversers/dfu.cpp

```diff
@@ -310,6 +310,11 @@ int dfu_traverser_t::run (Jobspec::Jobspec &jobspec,
         detail::dfu_impl_t::update ();
     } else if ( (rc = schedule (jobspec, meta, x, op, root, dfv)) ==  0) {
         *at = meta.at;
+        if (*at == graph_end) {
+            // no schedulable point found even at the end of the time, return EBUSY
+            errno = EBUSY;
+            return -1;
+        }
         if (*at < 0 or *at >= graph_end) {
```

Comment:
I don't think this really matters, but `at == graph_end` satisfies both the `if` condition above and this one.
```suggestion
        if (*at < 0 or *at > graph_end) {
```